src/main/java/org/broadinstitute/hellbender/utils/io/Resource.java

```diff
@@ -84,13 +84,16 @@ public InputStream getResourceContentsAsStream() {
      */
     public static File getResourceContentsAsFile(final String resourcePath) throws IOException {
         final File tmpResourceFile = File.createTempFile("tmp_read_resource_", ".config");
-        final InputStream systemResourceAsStream = ClassLoader.getSystemResourceAsStream(resourcePath);
+        InputStream resourceAsStream = ClassLoader.getSystemResourceAsStream(resourcePath);
 
-        if (systemResourceAsStream == null) {
-            throw new GATKException("Null value when trying to read system resource.  Cannot find: " + resourcePath);
+        if (resourceAsStream == null) {
```

Comment:
Can you add a one-line comment why we need this?  "For Spark, etc etc"

Reply:
Done